### DISCUSSION TITLE
Set Player NameTag

### DISCUSSION THREAD

**Discussion by FloGamerMC** [Minestom/Minestom#1066] [Help]
I tryed to use `Player#setCustomName`  but it dosn't work. How can I change the NameTag?

**Comment by PlugDev-class** [comment]
You could set the display-name with Player#setDisplayName
Theoretically this'd the job

**Reply by FloGamerMC** [reply]
https://apfelm.us/dmtgc I set it but the name was only set in the tablist

**Comment by emortaldev** [comment]
One way is to set their team. Teams have prefixes, suffixes and name colours (though name colours don't have custom hex).
(I don't know if this is the only way)
You then would use player.setDisplayName for the tablist
